tools/misbehaving-jmx-server/Dockerfile

```
@@ -0,0 +1,27 @@
+# Use the official Maven image as the base image
+FROM maven:latest
+
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy the pom.xml file and install the dependencies
+COPY pom.xml .
+RUN mvn dependency:resolve
+
+# Copy the source code and build the JAR file
+COPY src/ /app/src/
+RUN mvn clean package assembly:single
+
+RUN ls /app/target
+
+# Use the official OpenJDK image as the base image for the final image
+FROM openjdk:latest
```

```suggestion
FROM openjdk:latest AS final
```
Or some other name.

As this is just a test container, you could skip the `assembly:single` and `package` steps and just go straight to having some Maven target that runs the test server. I guess if we think we will use it somewhere else then it might be good to have a slim image that we publish.